From 065a57443c58a01c2abb77148587326562c52937 Mon Sep 17 00:00:00 2001
From: "Tang, Haoyu" <haoyu.tang@intel.com>
Date: Mon, 29 Oct 2018 17:20:17 +0800
Subject: [PATCH 0873/1214] ablbc: add target "adb" & "shell" for Android P

Google added "adb" or "shell" to reboot reason if there is no
arguments after "reboot" command on Android P. So need to add
"adb" and "shell" into ablbc boot target.

Tracked-On: PKT-1553 OAM-69459
Change-Id: I1e820e97dd5f7f10a0d443bc8652735b05d955f4
Signed-off-by: Tang, Haoyu <haoyu.tang@intel.com>
---
 drivers/staging/android/abl/ablbc.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/staging/android/abl/ablbc.c b/drivers/staging/android/abl/ablbc.c
index 59154f5..86cbd8d 100644
--- a/drivers/staging/android/abl/ablbc.c
+++ b/drivers/staging/android/abl/ablbc.c
@@ -94,6 +94,8 @@ struct nvram_msg {
 static const struct name2id NAME2ID[] = {
 	{ "main",	0x00 },
 	{ "android",	0x00 },
+	{ "adb",        0x00 },
+	{ "shell",      0x00 },
 	{ "bootloader",	0x01 },
 	{ "fastboot",	0x01 },
 	{ "elk",	0x02 },
-- 
2.7.4

